SEMANTICS.md

```diff
@@ -5,14 +5,41 @@ in case of an error, the resulting HedyException.
 
 ## Level 1
 
-### Commands
+### Commands and types
 
 Level 1 supports:
-* `print`
-* `ask`
-* `forward`
-* `turn`
-* `echo`
+
+<table>
+<thead>
+  <tr>
+    <th>Command</th>
+    <th>Types</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>print</td>
+    <td>integer | string</td>
+  </tr>
+  <tr>
+    <td>ask</td>
+    <td>integer | string</td>
+  </tr>
+  <tr>
+    <td>forward</td>
+    <td>integer</td>
+  </tr>
+  <tr>
+    <td>turn</td>
+    <td>'right' + integer | 'left' + integer</td>
+  </tr>
+  <tr>
+    <td>echo</td>
+    <td>any?</td>
```

Comment:
This can just be int + string I think, as we are doing for print? Because it supports the same operations. We can also do any for print of course, but it would be nicer to use the same.